app/views/account/pages/index.html.slim

```diff
@@ -2,6 +2,11 @@
   .row
     .col-md-12
       h1 All company pages
+  .row.breadcrumbs-top
+    .breadcrumb-wrapper.col-12
```

Comment:
`.col-12` ?
also, always put bootstrap classes in the beginning of set of classes